docker/Dockerfile.dev

```diff
@@ -25,13 +25,13 @@ ENV ENV_FILE .env.dev
 
 WORKDIR /usr/src/app
 
-COPY b3lb/requirements.txt \
-     b3lb/manage.py \
+COPY b3lb/requirements.txt .
+RUN pip3 install --no-cache-dir --prefix=/usr/local -r requirements.txt
+COPY b3lb/manage.py \
```

Comment:
Please do not add another container image layer if not required.

Reply:
It's used to build the image faster for development deployment, before this change, every time a code change occurs, all modules had to be downloaded and installed.